content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_control-plane.md

```diff
@@ -71,23 +71,3 @@ kubeadm init phase control-plane [flags]
   </tbody>
 </table>
 
-
-
-<!-- 
```

Comment:
IMO, one PR better has one purpose. The `see also` issue seems to have its own tracking issue.

Reply:
/lgtm
for this change 
I remember that  this issue (removing SEE ALSO ) has been closed

Reply:
see https://github.com/kubernetes/website/issues/18411